perun-core/src/main/java/cz/metacentrum/perun/core/bl/ExtSourcesManagerBl.java

```diff
@@ -19,6 +19,7 @@
 import cz.metacentrum.perun.core.api.exceptions.ExtSourceNotExistsException;
 import cz.metacentrum.perun.core.api.exceptions.ExtSourceUnsupportedOperationException;
 import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.PrivilegeException;
```

Comment:
Do not use Privilege Exception in other layers then Entry! This one is just for testing privileges in the entry layer for users using RPC of perun.